plugins/base/src/main/kotlin/parsers/Parser.kt

```diff
@@ -49,10 +51,51 @@ abstract class Parser {
 
     private fun jkdocToListOfPairs(javadoc: String): List<Pair<String, String>> =
```

Comment:
let's rename this to `parseTags` or something like this?

I've run dokkaHtml on kotlinx.coroutines and dumped the contents of the `javadoc` argument - it's all `module.md` contents, there's not a single kdoc/javadoc. The method and the argument names are really misleading

`parseTags` is a generic name that can be used for both k/jdoc and `module.md` files

Reply:
> let's rename this to `parseTags` or something like this?

It does not parse tags entirely. Besides, we already have `parseTagWithBody` method.

> it's all module.md contents, there's not a single kdoc/javadoc. 

It may be not only the multi-module documentation, but module and package documentation as well. 
In first case the method parse the all `module.md` contents.

Reply:
The naming is up to you, but `jkdoc` part should definitely be removed, it's misleading